testing/kubernetes/run-controlplane-kubernetes.sh

```diff
@@ -56,25 +52,6 @@ if [ "$1" == "start" ]; then
         sleep 10s
     done
 
-    # Wait for registry route to set up
-    echo "Waiting for registry route to set up"
-    attempt=0
-    while true; do
-        code=$(curl -w "%{http_code}" --max-time 10 "${REGISTRY_URL}/uptime" -o /dev/null)
-        if [ "$code" = "200" ]; then
-            echo "Registry route is set to '$REGISTRY_URL'"
-            break
-        fi
-
-        attempt=$((attempt + 1))
-        if [ "$attempt" -gt 10 ]; then
-            echo "Timeout waiting for registry route: /uptime returned HTTP ${code}"
-            echo "Deploying the controlplane has failed"
-            exit 1
-        fi
-        sleep 10s
-    done
-
```

Comment:
At some point, we should completely get rid of the run-controlplane.sh scripts. Just like the examples. We can make do easily with `kubectl create -f` and `docker-compose up -d`